### ISSUE TITLE
Got 500 Internal Server Error

### ISSUE THREAD

**Issue by x2c2mega** [org-roam/org-roam-server#106]
I'm using Doom Emacs with Emacs 27.1 version.

My org-roam-directory has sub-directories, but currently only one sub-directory has notes. My notes were rendered visually fine a couple of days back, but now I don't know why it's all blank now. Maybe after I did latest updates, but not sure. From Chrome console debug, I see this error message:

**HTTP request:**
http://127.0.0.1:8099/roam-data?force=1&token=null

**HTTP response:**
```
<!DOCTYPE html>
<html><head>
<title>500 Internal Error</title>
</head><body>
<h1>500 Internal Error</h1>
<p>Internal error when handling this request.</p>
<pre>error: (emacsql-error near line 83: no such column: dest)
</pre>
</body></html>
```

How do I go about fixing this issue or show me to debug further? Thanks.

**Comment by goktug97** [comment]
Can you update `org-roam`

**Comment by x2c2mega** [comment]
Thanks for link to issue that was fixed regarding changing columns to avoid conflicts with SQLite reserved keywords. I checked Doom Emacs' local cache for org-roam and the fix wasn't there. Unfortunately, updating locally via Git pull didn't work, because Doom Emacs pulled older version again when trying to build/compile it. After deleting the org-roam repo and doing another sync, I saw it's pulling from emacsmirror-mirror site which probably doesn't have latest changes from org-roam master yet. I'll wait for a couple more days. I'll go ahead and close the ticket then. Thanks for the awesome package!
